packages/wrangler/src/deployment-bundle/bundle.ts

```diff
@@ -317,12 +362,12 @@ export async function bundleWorker(
 				...define,
 			},
 		}),
-		alias,
 		loader: {
 			...COMMON_ESBUILD_OPTIONS.loader,
 			...(loader || {}),
 		},
 		plugins: [
+			aliasPlugin,
```

Comment:
what's the order in which these plugins are executed? Is first to last in the `plugins` array, or the other way around? I can't find that mentioned in the esbuild docs

Reply:
First to last. I confirmed by playing with my repro linked in the issue.

Reply:
in that case, what happens if `aliasPlugin` and the unenv one are trying to alias the same module? In that case, won't unenv override the user alias?

Reply:
Good question. esbuild runs the onResolve hooks on each plugin in order **until** one does not return undefined. It doesn't run anymore onResolve hooks once it receives a path.

Reply:
TIL! thx for clarifying that 🙏

Reply:
not sure if we want to capture this use case in a test, I'll leave that up to you